crates/ingress-rpc/src/lib.rs

```diff
@@ -181,7 +181,8 @@ pub struct Config {
 
 pub fn connect_ingress_to_builder(
     metering_rx: broadcast::Receiver<MeterBundleResponse>,
-    backrun_rx: broadcast::Receiver<tips_core::Bundle>,
+    backrun_rx: broadcast::Receiver<(tips_core::Bundle, uuid::Uuid)>,
+    tx_bundle_id_rx: broadcast::Receiver<(TxHash, uuid::Uuid)>,
```

Comment:
i might be missing something, but is it possible to only keep `backrun_rx`?

my thinking is that `tips_core::Bundle` contains a `txs` field which is a `Vec<Bytes>`

if we were to do that then we'd probably need an enum/flag to decide on which `base_` RPC call to do